typespec-tests/tsp/examples/2022-06-01-preview/FlattenOp_SendLong.json

```diff
@@ -4,7 +4,7 @@
   "parameters": {
     "name":  "myRequiredId",
     "filter": "name=myName",
-    "request": {
+    "sendLongRequest": {
```

Comment:
TCGC parameter name would be `sendLongRequest`.

Also, the fix within this PR makes it work if this is called `body`.

Reply:
Yeah, the fix should solve the example issue, but I'm not sure if service know they should write in this way? `request` seems a common name, like `responses`, while `sendLongRequest` is a special name for each operation.

Reply:
So far (at least in mgmt), the `body` is actually the common name (and that's the reason we had code to fallback to `body`).

Reply:
Should we use `body` in the example here? Guess in the future, no service will use TCGC given name for example body name...

Reply:
OK, let me change it to "body".

PS: so far, the observation of what JSON example typically does is from Swagger. I'd assume things may change it source is TypeSpec (as it is less clear to service about what the parameter name be, if deduced from tsp across templates and anonymous models).

Reply:
Yeah, `body` is much better